packages/calcite-components/src/components/chip-group/chip-group.stories.ts

```diff
@@ -105,3 +105,79 @@ export const darkThemeRTL_TestOnly = (): string => html`
 `;
 
 darkThemeRTL_TestOnly.parameters = { themes: modesDarkDefault };
+
+export const interactivityStates_TestOnly = (): string => html`
+  <div style="width:500px; max-width:100%; background-color:white; padding:100px">
```

Comment:
This might not be the case, but let’s remove any non-essential inline styles for the test to keep it as focused and simple as possible.